test/NuGet.Core.Tests/NuGet.DependencyResolver.Core.Tests/RemoteDependencyWalkerTests.cs

```diff
@@ -1603,16 +1603,14 @@ public async Task CentralTransitiveDependency_Downgrades_IfDowngrades_WereNotRej
                     .DependsOn("F", version100);
             provider.Package("F", version100)
                     .DependsOn("C", version200);
-            provider.Package("C", version200)
-                    .DependsOn("H", version200);
 
-            // A -> -> G 1.0.0 -> H 2.0.0(this will not be rejected) -> D 1.0.0
+            // A -> -> G 1.0.0 -> H 2.0.0(this will not be rejected) -> D 2.0.0 (this will be downgraded)
```

Comment:
Here too for `H`.

Reply:
I don't see the issue here. 

H 2.0 depends on D 2.0